

<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
  <meta charset="utf-8" />
  
    <meta property="article:modified_time" content="2021-01-10T14:51:05-05:00" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  
  <title>Day 0: Software and training setup &mdash; Galloway Lab Protocols  documentation</title>
  

  
  <link rel="stylesheet" href="../../_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="../../_static/pygments.css" type="text/css" />
  <link rel="stylesheet" href="../../_static/css/custom.css" type="text/css" />

  
  

  
  

  

  
  <!--[if lt IE 9]>
    <script src="../../_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
    
      <script type="text/javascript" id="documentation_options" data-url_root="../../" src="../../_static/documentation_options.js"></script>
        <script src="../../_static/jquery.js"></script>
        <script src="../../_static/underscore.js"></script>
        <script src="../../_static/doctools.js"></script>
        <script async="async" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/latest.js?config=TeX-AMS-MML_HTMLorMML"></script>
    
    <script type="text/javascript" src="../../_static/js/theme.js"></script>

    
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" />
    <link rel="next" title="An introduction to Git" href="git_intro.html" />
    <link rel="prev" title="Tissue culture basics" href="TC.html" /> 
</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">
    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search"  style="background: #2c6854" >
          

          
            <a href="../../index.html" class="icon icon-home"> Galloway Lab Protocols
          

          
          </a>

          
            
            
              <div class="version">
                latest
              </div>
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        
        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <ul class="current">
<li class="toctree-l1"><a class="reference internal" href="../../protocols/index.html">Protocols</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../../protocols/analytics/index.html">Analytical protocols</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../protocols/analytics/atac_see.html">ATAC-see</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/analytics/atac_see.html#transposase-buffer-preparation">Transposase buffer preparation</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/analytics/atac_see.html#id1">ATAC-see</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../protocols/analytics/attune_operation.html">Attune operation</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/analytics/attune_operation.html#startup">Startup</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/analytics/attune_operation.html#cleaning">Cleaning</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/analytics/attune_operation.html#performance-check">Performance Check</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/analytics/attune_operation.html#maintainence">Maintainence</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../protocols/analytics/single_cell_qpcr.html">Single cell qPCR</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/analytics/single_cell_qpcr.html#single-cell-acquisition">Single-cell acquisition</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/analytics/single_cell_qpcr.html#lysis-reverse-transcription-rt-and-specific-target-amplification-sta">Lysis, reverse transcription (RT), and specific target amplification (STA)</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/analytics/single_cell_qpcr.html#single-cell-qpcr-on-fluidigm-biomark">Single-cell qPCR on Fluidigm Biomark</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../protocols/cloning/index.html">Cloning protocols</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../protocols/cloning/gibson.html">Gibson assembly</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/cloning/gibson.html#protocol">Protocol</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../protocols/cloning/lr_cloning.html">LR cloning</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/cloning/lr_cloning.html#protocol">Protocol</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/cloning/lr_cloning.html#expected-results">Expected results</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../protocols/cloning/transformation.html">Bacterial transformation</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/cloning/transformation.html#cloning-summary">Cloning summary</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/cloning/transformation.html#heat-shock-transformation-chemically-competent-bacteria">Heat-shock transformation (chemically competent bacteria)</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../protocols/protein_production/index.html">Protein production</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../protocols/protein_production/bis_tris_protein_gels.html">Protein gel casting</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/protein_production/bis_tris_protein_gels.html#required-stock-solutions">Required stock solutions</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/protein_production/bis_tris_protein_gels.html#casting-protocol">Casting protocol</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/protein_production/bis_tris_protein_gels.html#gel-casting-setup">Gel casting setup</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../protocols/protein_production/chitin_protein_purification.html">Chitin-binding protein purification</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/protein_production/chitin_protein_purification.html#required-solutions">Required solutions</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/protein_production/chitin_protein_purification.html#protocol">Protocol</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/protein_production/chitin_protein_purification.html#regeneration-protocol">Regeneration Protocol</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../protocols/protein_production/denaturing_protein_gel.html">Denaturing protein gel run and staining</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/protein_production/denaturing_protein_gel.html#solutions-required">Solutions required</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/protein_production/denaturing_protein_gel.html#running-procedure">Running procedure</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/protein_production/denaturing_protein_gel.html#staining">Staining</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../protocols/protein_production/protein_expression.html">Protein expression</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/protein_production/protein_expression.html#protocol">Protocol</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../protocols/protein_production/protein_purification.html">His-tag protein purification</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/protein_production/protein_purification.html#required-solutions">Required solutions</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/protein_production/protein_purification.html#protocol">Protocol</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/protein_production/protein_purification.html#id1">Protocol</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../protocols/tc/index.html">TC protocols</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../protocols/tc/adherent_cell_culture.html">Adherent Cell Culture</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/tc/adherent_cell_culture.html#general-culture-handling-procedure">General culture handling procedure</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/tc/adherent_cell_culture.html#changing-media">Changing media</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/tc/adherent_cell_culture.html#passaging-cells">Passaging cells</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../protocols/tc/adherent_staining.html">Adherent Cell Staining</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/tc/adherent_staining.html#protocol">Protocol</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../protocols/tc/cryopreservation.html">Cryopreservation of mammalian cells</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/tc/cryopreservation.html#cryopreservation-medium-should-always-be-made-fresh">Cryopreservation Medium (should always be made fresh)</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../protocols/tc/good_practices.html">Important Tissue Culture Practices</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../protocols/tc/media.html">Tissue Culture Media</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/tc/media.html#fetal-bovine-serum-fbs-aliquots">Fetal Bovine Serum (FBS) Aliquots</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/tc/media.html#pen-strep-aliquots">Pen-strep aliquots</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/tc/media.html#preparation-of-basic-2d-cell-culture-media-dmem-10-fbs-1-pen-strep">Preparation of basic 2D cell culture media ( DMEM + 10% FBS +1% Pen-strep)</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../protocols/tc/plat_e_transfection.html">Plat-E Retroviral Transduction of MEFs</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/tc/plat_e_transfection.html#plat-e-transfection">Plat-E Transfection</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/tc/plat_e_transfection.html#transduction-of-mouse-embryonic-fibroblasts-mefs">Transduction of Mouse Embryonic Fibroblasts (MEFs)</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../protocols/tc/transient_transfection.html">Transient 293T Transfection</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/tc/transient_transfection.html#protocol">Protocol</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../protocols/tc/virus_concentration.html">Virus concentration</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../protocols/tc/virus_concentration.html#protocol">Protocol</a></li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../../recipes/index.html">Recipes</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../../recipes/bacteria/index.html">Bacterial recipes</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../recipes/bacteria/antibiotic_stock.html">Antibiotic Stock Recipes</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/bacteria/antibiotic_stock.html#x-ampicillin-amp">1000X Ampicillin (Amp)</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/bacteria/antibiotic_stock.html#x-kanamycin-kan">1000X Kanamycin (Kan)</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/bacteria/antibiotic_stock.html#x-chloramphenicol-chlor">1000X Chloramphenicol (Chlor)</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../recipes/bacteria/media.html">Bacterial media recipes</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/bacteria/media.html#tb-terrific-broth">TB (Terrific Broth)</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/bacteria/media.html#x-tb-salts">10x TB salts</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../recipes/tc/index.html">TC recipes</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../recipes/tc/media.html">TC media recipes</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/media.html#n3-media">N3 media</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/media.html#mef-media">MEF media</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/media.html#breast-cancer-1-media">Breast cancer 1 media</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/media.html#breast-cancer-2-media">Breast cancer 2 media</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/media.html#breast-cancer-3-media">Breast cancer 3 media</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/media.html#breast-cancer-4-media">Breast cancer 4 media</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/media.html#glia-media">Glia media</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/media.html#sorting-media">Sorting media</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/media.html#collection-media">Collection media</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/media.html#dissociation-media">Dissociation media</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/media.html#freezing-media">Freezing media</a></li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../../tech/index.html">Tech documentation</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../../tech/repo_setup.html">Repository setup</a></li>
</ul>
</li>
<li class="toctree-l1 current"><a class="reference internal" href="../index.html">Bootcamp</a><ul class="current">
<li class="toctree-l2 current"><a class="reference internal" href="index.html">IAP Bootcamp</a><ul class="current">
<li class="toctree-l3"><a class="reference internal" href="TC.html">Tissue culture basics</a></li>
<li class="toctree-l3 current"><a class="current reference internal" href="#">Day 0: Software and training setup</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#ehs-setup-and-trainings">EHS setup and trainings</a></li>
<li class="toctree-l4"><a class="reference internal" href="#software-and-webservices">Software and webservices</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="git_intro.html">An introduction to Git</a><ul>
<li class="toctree-l4"><a class="reference internal" href="git_intro.html#motivation">Motivation</a></li>
<li class="toctree-l4"><a class="reference internal" href="git_intro.html#git-by-example">Git by example</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="software_tips_and_tricks.html">Software Tips and Tricks</a><ul>
<li class="toctree-l4"><a class="reference internal" href="software_tips_and_tricks.html#one-date-format-to-rule-them-all-naming">One date format to rule them all/naming</a></li>
<li class="toctree-l4"><a class="reference internal" href="software_tips_and_tricks.html#paper-rss-feeds">Paper RSS feeds</a></li>
<li class="toctree-l4"><a class="reference internal" href="software_tips_and_tricks.html#creating-feeds-from-searches">Creating feeds from searches</a></li>
<li class="toctree-l4"><a class="reference internal" href="software_tips_and_tricks.html#zotero">Zotero</a></li>
<li class="toctree-l4"><a class="reference internal" href="software_tips_and_tricks.html#better-quartzy">Better Quartzy</a></li>
<li class="toctree-l4"><a class="reference internal" href="software_tips_and_tricks.html#regex-help">Regex help</a></li>
<li class="toctree-l4"><a class="reference internal" href="software_tips_and_tricks.html#fonts">Fonts</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="the_shell.html">On Terminals and Shells</a><ul>
<li class="toctree-l4"><a class="reference internal" href="the_shell.html#motivation">Motivation</a></li>
<li class="toctree-l4"><a class="reference internal" href="the_shell.html#the-basics">The basics</a></li>
<li class="toctree-l4"><a class="reference internal" href="the_shell.html#interface-basics">Interface basics</a></li>
<li class="toctree-l4"><a class="reference internal" href="the_shell.html#starting-off-at-home">Starting off at home</a></li>
<li class="toctree-l4"><a class="reference internal" href="the_shell.html#moving-away-from-home">Moving away from home</a></li>
<li class="toctree-l4"><a class="reference internal" href="the_shell.html#relative-and-absolute-paths">Relative and absolute paths</a></li>
<li class="toctree-l4"><a class="reference internal" href="the_shell.html#file-operations">File operations</a></li>
<li class="toctree-l4"><a class="reference internal" href="the_shell.html#editing-and-viewing-files">Editing and viewing files</a></li>
<li class="toctree-l4"><a class="reference internal" href="the_shell.html#finding-things-in-documents">Finding things in documents</a></li>
<li class="toctree-l4"><a class="reference internal" href="the_shell.html#exercise">Exercise</a></li>
<li class="toctree-l4"><a class="reference internal" href="the_shell.html#extras">Extras</a></li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../../contributor_guide.html">Contributor guide</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../../contributor_guide.html#local-environment-setup">Local environment setup</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../contributor_guide.html#text-editor">Text editor</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../contributor_guide.html#python-sphinx">Python/Sphinx</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../contributor_guide.html#git">git</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../contributor_guide.html#local-building">Local building</a></li>
<li class="toctree-l2"><a class="reference internal" href="../../contributor_guide.html#online-editing-through-github">Online editing through Github</a></li>
<li class="toctree-l2"><a class="reference internal" href="../../contributor_guide.html#repository-layout">Repository layout</a></li>
<li class="toctree-l2"><a class="reference internal" href="../../contributor_guide.html#basics-of-restructuredtext">Basics of reStructuredText</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../contributor_guide.html#simple-markup">Simple markup</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../contributor_guide.html#explicit-markup">Explicit markup</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../contributor_guide.html#admonitions">Admonitions</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../contributor_guide.html#code">Code</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../contributor_guide.html#tables">Tables</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../contributor_guide.html#references-and-links">References and links</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../contributor_guide.html#images">Images</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../contributor_guide.html#math">Math</a></li>
</ul>
</li>
</ul>
</li>
</ul>

            
          
        </div>
        
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="../../index.html">Galloway Lab Protocols</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          

















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="../../index.html" class="icon icon-home"></a> &raquo;</li>
        
          <li><a href="../index.html">Bootcamp</a> &raquo;</li>
        
          <li><a href="index.html">IAP Bootcamp</a> &raquo;</li>
        
      <li>Day 0: Software and training setup</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
          
            <a href="../../_sources/bootcamp/iap/day_0_setup.rst.txt" rel="nofollow"> View page source</a>
          
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="day-0-software-and-training-setup">
<h1>Day 0: Software and training setup<a class="headerlink" href="#day-0-software-and-training-setup" title="Permalink to this headline">¶</a></h1>
<p>Our lab uses a lot of different software and various webservices.
Before beginning IAP, you should install at least everything required
on the list here, as well as completing all of the required EHS trainings.</p>
<div class="section" id="ehs-setup-and-trainings">
<h2>EHS setup and trainings<a class="headerlink" href="#ehs-setup-and-trainings" title="Permalink to this headline">¶</a></h2>
<p>Adding yourself to the lab’s training group will register you with the
EHS system and add all necessary trainings to your profile.</p>
<ol class="arabic">
<li><p>Go to <a class="reference external" href="https://atlas.mit.edu">https://atlas.mit.edu</a> and go to the learning center, through the tab
on the left:</p>
<img alt="Learning center" class="align-center" src="../../_images/atlas_learning_center.png" />
</li>
<li><p>In the upper right, select “My profile”, then “Update PI/activities”</p></li>
<li><p>Add Kate E. Galloway as your PI.</p></li>
<li><p>Select the following training types. If you are an undergrad, do not
select the BL2+ training group.</p></li>
</ol>
<img alt="Select the BL1/BL2 and BL2+ training categories, plus then performing research with human cells." src="../../_images/atlas_biosafety_training.png" />
<img alt="Select the 'use potentially hazardous chemicals&quot;." src="../../_images/atlas_chemical_training.png" />
<img alt="Select the &quot;working with cryogenic liquids&quot; training." src="../../_images/atlas_cryo_training.png" />
<ol class="arabic simple" start="5">
<li><p>After submitting, you will see many required trainings. Some have a
required ‘classroom’ component, such as the <em>Lab Specific Chemical Hygiene
training</em>, which will be completed with an in-lab walkaround.</p></li>
</ol>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p>One of the components of the bloodborne pathogen training
is the opportunity to be vaccinated for Hepatitis B or to
have an antibody titer test for free.</p>
<p>Most of us were vaccinated for HepB as children, but that
vaccine was only ~90% effective, so you may want to get
the free antibody titer test. You can get a free booster
or get doses of a new, more modern HepB vaccine if you
no longer have HepB antibodies.</p>
</div>
<ol class="arabic simple" start="6">
<li><p>If you are going to be helping with mouse work, still under the
‘My Profile’ tab, you should click <strong>Join Another Group</strong> under training
groups and add the <strong>68N: Mouse</strong> training group.</p></li>
</ol>
</div>
<div class="section" id="software-and-webservices">
<h2>Software and webservices<a class="headerlink" href="#software-and-webservices" title="Permalink to this headline">¶</a></h2>
<div class="section" id="core-webservices">
<h3>Core webservices<a class="headerlink" href="#core-webservices" title="Permalink to this headline">¶</a></h3>
<ul class="simple">
<li><p>Create a <a class="reference external" href="https://www.quartzy.com/">Quartzy account</a>. Using your MIT email is recommended.</p></li>
<li><p>Create a <a class="reference external" href="https://github.com/">Github account</a>. You can use either a personal or MIT email.</p>
<ul>
<li><p>Activate your student benefits by going to <a class="reference external" href="https://education.github.com/discount_requests/student_application">https://education.github.com/discount_requests/student_application</a>.
You will be asked to connect your MIT email and send in a picture of your student ID. (Because
MIT does not remove emails for alums, they need to confirm active student status.)</p></li>
</ul>
</li>
<li><p>Create a <a class="reference external" href="https://www.zotero.org/user/register">Zotero account</a>. Using a personal email is
recommended for permanence reasons.</p></li>
<li><p>Create an <a class="reference external" href="https://orcid.org/register">ORCID</a>. Adding all of your active emails is recommended.</p></li>
</ul>
<p>After creating these accounts, message your <strong>Kerberos ID</strong>, your <strong>Github username</strong>, your <strong>Zotero username</strong>,
and the email you used to create your <strong>Quartzy account</strong> to the <code class="docutils literal notranslate"><span class="pre">#iap</span></code> slack channel. We’ll get you added to
all of the above. You will have to:</p>
<ul class="simple">
<li><p>Accept the Github invitation to the <a class="reference external" href="https://github.com/gallowaylabmit">gallowaylabmit organization</a>.</p></li>
<li><p>Accept the Zotero invitation to the <code class="docutils literal notranslate"><span class="pre">gallowaylab</span></code> group,
checking that it appears in your <a class="reference external" href="https://www.zotero.org/groups/">group list</a>.</p></li>
</ul>
</div>
<div class="section" id="shared-storage">
<h3>Shared storage<a class="headerlink" href="#shared-storage" title="Permalink to this headline">¶</a></h3>
<p>We use <strong>OneDrive</strong> for shared lab storage. OneDrive’s web interface is slightly clunky,
and occasionally you have to wait a minute for syncing to occur, but it is the MIT option
that offers us the largest amount of storage space, without having bandwidth limits (unlike MIT’s Google Drive).</p>
<p>OneDrive also has an excellent implementation of “files-on-demand”/”online sync”, where all
files in the shared storage <em>appear</em> to be accessible, but do not actually take up local
disk space until you open them/unless you manually trigger a download, at which point
the software invisibly downloads files in the background. There’s no cost to having
the entire shared folder locally synced.</p>
<p>OneDrive also has tight integration with Office products, allowing Google Drive-esque live,
multi-person editing of Office documents saved inside it.</p>
<p>We share presentations, software, primers, and plasmids with each other through the shared
library, and all lab instruments will save data into OneDrive so everything is accessible.</p>
<p>When you see a file location like <code class="docutils literal notranslate"><span class="pre">instruments\data\attune</span></code> without an additional information,
it is likely a path inside the OneDrive.</p>
<p>After being given access:</p>
<ol class="arabic">
<li><p>If not on a recent version of Windows 10 (e.g. Win 7, MacOSX, etc), download the
<a class="reference external" href="https://www.microsoft.com/en-us/microsoft-365/onedrive/download">OneDrive client</a>.
Recent versions of Windows come with this preinstalled.</p></li>
<li><p>Bookmark the web version here: <a class="reference external" href="https://mitprod.sharepoint.com/sites/GallowayLab/Shared%20Documents">https://mitprod.sharepoint.com/sites/GallowayLab/Shared%20Documents</a></p></li>
<li><p>On the web version, select the <strong>Sync</strong> button in the top tab:</p>
<blockquote>
<div><img alt="The sync button is the fourth button across." src="../../_images/onedrive_sync.png" />
</div></blockquote>
</li>
<li><p>This will trigger the OneDrive software you installed. It will ask you for a local folder
to sync into. After several minutes, it will show “OneDrive is up to date”, and all files
should be accessible.</p></li>
</ol>
</div>
<div class="section" id="experimental-software">
<h3>Experimental software<a class="headerlink" href="#experimental-software" title="Permalink to this headline">¶</a></h3>
<ul class="simple">
<li><p><strong>SnapGene:</strong> We use SnapGene for our cloning analysis. Download it through IST
<a class="reference external" href="https://downloads.mit.edu/released/snapgene/vendor-registration.html">here</a>,
if asked for a registration code, get it
<a class="reference external" href="http://downloads.mit.edu/released/snapgene/group-name_registration-code.txt">here</a>
(certificates required for both links).</p></li>
<li><p><strong>FlowJo:</strong> When initially analyzing flow cytometery data, you can install a FlowJo demo from <code class="docutils literal notranslate"><span class="pre">instruments\software</span></code>.
The demo is limited time, so don’t install it until you have data to analyze!</p></li>
<li><p><strong>FIJI:</strong> For simple image analysis, Fiji (ImageJ) gives a nice GUI interface. Download it from <a class="reference external" href="https://fiji.sc">https://fiji.sc</a></p></li>
<li><p><strong>CellProfiler:</strong> CellProfiler is an excellent tool for doing image cytometry (analyzing cell-by-cell in image data).
In contrast to the GUI-only tools built into the Keyence software, CellProfiler enables repeatable, pipelinable analyses.
Download it from <a class="reference external" href="https://cellprofiler.org/">https://cellprofiler.org/</a></p></li>
</ul>
</div>
<div class="section" id="other">
<h3>Other<a class="headerlink" href="#other" title="Permalink to this headline">¶</a></h3>
<ul>
<li><p><strong>Zotero:</strong> Zotero is an excellent free, open-source citation manager. After downloading Zotero from <a class="reference external" href="https://www.zotero.org/">https://www.zotero.org/</a>,
it should prompt you to install the Zotero Connector, a browser plugin that lets you download paper citations with one-click.
If not, download the connector <a class="reference external" href="https://www.zotero.org/download/connectors">here</a>. We also have a shared Zotero group, to
accumulate citations when writing manuscripts.</p>
<p>Several helpful plugins can be downloaded; the recommended ones are:</p>
<table class="docutils align-default" id="id1" style="width: 100%">
<caption><span class="caption-text">Recommended Zotero plugins</span><a class="headerlink" href="#id1" title="Permalink to this table">¶</a></caption>
<colgroup>
<col style="width: 50%" />
<col style="width: 50%" />
</colgroup>
<thead>
<tr class="row-odd"><th class="head"><p>Addon name</p></th>
<th class="head"><p>Description</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p><a class="reference external" href="http://zotfile.com/">ZotFile</a></p></td>
<td><p>Enables useful file operations, such as extracting annotations from a marked-up PDF,
transferring new papers to a tablet for annotation, and auto-file renaming.</p></td>
</tr>
<tr class="row-odd"><td><p><a class="reference external" href="https://github.com/wshanks/Zutilo">Zutilo</a></p></td>
<td><p>Enables helpful tagging operations, such as the ability to copy/paste tags or easily add paper relationships.</p></td>
</tr>
<tr class="row-even"><td><p><a class="reference external" href="https://retorque.re/zotero-better-bibtex/">Better Bibtex</a></p></td>
<td><p>If you plan to use LaTeX, install this plugin before exporting to BibTeX. This addon makes nice-looking,
stable citation keys that do not change on export.</p></td>
</tr>
</tbody>
</table>
</li>
</ul>
<div class="admonition-downloading-zotero-plugins-through-firefox admonition">
<p class="admonition-title">Downloading Zotero plugins through Firefox</p>
<p>Since Zotero is built on modified Firefox, Zotero plugins appear similar to Firefox plugins. If downloading
these plugins through Firefox, you will need to explicitly right click-&gt;download target; left-clicking on download
links will attempt to install the Zotero plugin as a Firefox plugin, which will fail.</p>
</div>
<ul>
<li><p><strong>Creative Cloud:</strong>  MIT recently opened Adobe’s Creative Cloud to all students. After installing the
<a class="reference external" href="https://www.adobe.com/creativecloud/desktop-app.html">Creative Cloud application</a>, login with your MIT credentials,
after selecting “Work/School account”. You may have to wait 24 hours for activation after your first login. You should
install <strong>Acrobat</strong> (for viewing PDFs) and <strong>Illustrator</strong> (for drawing vector art).</p></li>
<li><p><strong>Inkscape:</strong> <em>(Optional)</em> Inkscape is a free and open-source vector drawing program that can be downloaded
<a class="reference external" href="https://inkscape.org/release/inkscape-1.0.1/">here</a>. Inkscape and Illustrator have many similar, but not completely overlapping features.
If you have not learned to use either, pick one to start with to learn first (likely Illustrator), However, it’s likely eventually
worth learning both if you don’t want to eventually pay for Creative Cloud. Inkscape’s (Cairo) PDF import also tends to be superior,
if trying to import vector images from paper PDFs.</p></li>
<li><p><strong>Color palletes:</strong> Having nice color-blind friendly, distinct colors to start drawings from is helpful.
If making cartoon/stylized figures, the colors in-pallette might be enough! If not, the pallette can provide
a good starting place.</p>
<p>You can download pre-created pallettes for both <a class="reference external" href="../../_static/iap_files/cat20_colors.ase">Illustrator</a>
and <a class="reference external" href="../../_static/iap_files/cat20_colors.gpl">Inkscape</a>
for the well-known Category20/20b color set, which is color-blind friendly (and becoming the default in more and more
software packages):</p>
<img alt="../../_images/illustrator_swatches.png" class="align-center" src="../../_images/illustrator_swatches.png" />
<p>To use these pallette files, see the <a class="reference external" href="https://helpx.adobe.com/illustrator/using/using-creating-swatches.html#share_swatches_between_applications">Illustrator documentation</a>
or the <a class="reference external" href="https://inkscape-manuals.readthedocs.io/en/latest/palette.html">Inkscape documentation</a>.</p>
</li>
</ul>
</div>
<div class="section" id="coding-and-collaboration">
<h3>Coding and collaboration<a class="headerlink" href="#coding-and-collaboration" title="Permalink to this headline">¶</a></h3>
<ul>
<li><p><strong>Slack</strong> is how we communicate! After <a class="reference external" href="https://slack.com/downloads">downloading it</a>, sign into
<a class="reference external" href="https://gallowaylab.slack.com">https://gallowaylab.slack.com</a>. In addition to the default channels, you may want to join <code class="docutils literal notranslate"><span class="pre">#sequencing</span></code> to get
your sequencing orders delivered right to you via Slack and join <code class="docutils literal notranslate"><span class="pre">#memes</span></code> for obvious reasons.</p></li>
<li><p><strong>Git:</strong> For any code/code-like thing (LaTeX, other plain-text files) you write, Git is the standard way to share
and collaborate with others in addition to tracking your entire version history.</p>
<p>You must install the base command-line tools from <a class="reference external" href="https://git-scm.com/downloads">here</a>, make sure that you select
your operating system and not the “Download source code” button!</p>
<div class="admonition tip">
<p class="admonition-title">Tip</p>
<p>When installing Git, you may want to change Git’s default editor to something other than Vim, such as VS Code.</p>
<p>When asked about adjusting the PATH environment, choose the <strong>Git from the command line and also from 3rd-party software</strong>
option; this makes sure all the other software also has Git access. All other defaults are fine, but can be changed
if you want.</p>
</div>
<p>After installation, you should set your global identity on that computer (e.g. what name/email gets stored alongside the work you do).</p>
<p>Open a terminal (Terminal on MacOS, Powershell on Windows) and type the following lines (without the beginning <code class="docutils literal notranslate"><span class="pre">$</span></code>, which identifies here that we are typing this into a terminal),
substituting your name and email (giving an email you associated with your Github account).</p>
<div class="highlight-console notranslate"><div class="highlight"><pre><span></span><span class="gp">$ </span>git config --global user.name <span class="s2">&quot;Full Name&quot;</span>
<span class="gp">$ </span>git config --global user.email email_address@example.com
</pre></div>
</div>
</li>
<li><p><strong>Github Desktop:</strong> This program is a good basic GUI Git tool, in case the command line interface/built in editor interfaces
aren’t for you. Download it <a class="reference external" href="https://desktop.github.com/">here</a>.</p></li>
<li><p><strong>Python:</strong> Python is an excellent “Jack of all trades” language; we use it extensively. If you are on MacOS, you may have
Python3 pre-installed; you can check by typing <code class="docutils literal notranslate"><span class="pre">python3</span></code> at a terminal. If you do not have Python preinstalled, you should
download it from <a class="reference external" href="https://www.python.org/downloads/">here</a>. Click the latest version download from the top, then scroll down
and select the 64-bit installer for your OS.</p>
<p>When installing, select <strong>Add Python to PATH</strong>; this ensures that when you type <code class="docutils literal notranslate"><span class="pre">python</span></code> at a terminal, you get this version you
just installed. Other software can also access this “default” installation.</p>
<div class="admonition-on-snakes-and-anaconda admonition">
<p class="admonition-title">On snakes and Anaconda</p>
<p>If you have Anaconda installed and don’t have an explicit reason to need it (e.g. conda-only packages),
it is recommended to uninstall Anaconda and install Python directly this way.</p>
<p>With modern Python, the benefits that Anaconda initially brought to the field (virtual environments
and pre-compiled packages) are now integrated into the normal Python ecosystem, making Anaconda
unnecessary. We also don’t want multiple Python versions competing.</p>
</div>
<p>After installing, to make sure it worked, open a new terminal and type the following, checking that the output looks
similar to the following.</p>
<div class="highlight-console notranslate"><div class="highlight"><pre><span></span><span class="gp">$ </span>python
<span class="go">Python 3.9.1 (tags/v3.9.1) [MSC v.1916 64 bit (AMD64)] on win32</span>
<span class="go">Type &quot;help&quot;, &quot;copyright&quot;, &quot;credits&quot; or &quot;license&quot; for more information.</span>
</pre></div>
</div>
<p>Exit the Python prompt by typing <code class="docutils literal notranslate"><span class="pre">exit()</span></code></p>
<p>After exiting Python, you can install the normal “semi-base” packages needed for most data analysis:</p>
<div class="highlight-console notranslate"><div class="highlight"><pre><span></span><span class="gp">$ </span>pip install numpy scipy pandas matplotlib jupyter jupyterlab
</pre></div>
</div>
</li>
<li><p><strong>R:</strong> Many bioinformatic tools are written in R, so we also use R. From <a class="reference external" href="http://lib.stat.cmu.edu/R/CRAN/">here</a>, download
the main package (MacOS) or both the <code class="docutils literal notranslate"><span class="pre">base</span></code> entry and the <code class="docutils literal notranslate"><span class="pre">Rtools</span></code> entry (Windows).</p></li>
<li><p><strong>VSCode:</strong> <em>(Optional)</em> Having a good <em>plain-text editor</em> (e.g., not Word) is important for coding, and is ultimately up to personal taste.
If you have your own favorite, feel free to not install VS Code. If you are used to language-specific IDE’s like MATLAB, IDLE,
or RStudio, VS Code allows you to do editing, debugging, previewing, source control, etc in a mostly language-agnostic manner;
once you customize it to how you want, you can use it for all of your coding.</p>
<p>The recommended editor is VS Code, downloadable <a class="reference external" href="https://code.visualstudio.com/">here</a>.</p>
<p>After installing, you should click the extensions button:</p>
<img alt="../../_images/vs_code_extensions.png" class="align-center" src="../../_images/vs_code_extensions.png" />
<p>and search and install the following extensions (type in the name, click the install button).</p>
<table class="docutils align-default" id="id2" style="width: 100%">
<caption><span class="caption-text">Recommended VS Code extensions</span><a class="headerlink" href="#id2" title="Permalink to this table">¶</a></caption>
<colgroup>
<col style="width: 33%" />
<col style="width: 33%" />
<col style="width: 33%" />
</colgroup>
<thead>
<tr class="row-odd"><th class="head"><p>Addon name</p></th>
<th class="head"><p>Image</p></th>
<th class="head"><p>Description</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>Python</p></td>
<td><p><a class="reference internal" href="../../_images/vs_code_python.png"><img alt="vsc_python" src="../../_images/vs_code_python.png" style="width: 200px;" /></a></p></td>
<td><p>Enables Python debugging, running, and IntelliSense (in-line help while typing).</p></td>
</tr>
<tr class="row-odd"><td><p>Pylance</p></td>
<td><p><a class="reference internal" href="../../_images/vs_code_pylance.png"><img alt="vsc_pylance" src="../../_images/vs_code_pylance.png" style="width: 200px;" /></a></p></td>
<td><p>Faster ‘language server’ for Python, which means the IntelliSense is faster and more accurate.</p></td>
</tr>
<tr class="row-even"><td><p>R</p></td>
<td><p><a class="reference internal" href="../../_images/vs_code_r.png"><img alt="vsc_r" src="../../_images/vs_code_r.png" style="width: 200px;" /></a></p></td>
<td><p>Base language support for R.</p></td>
</tr>
<tr class="row-odd"><td><p>R LSP Client</p></td>
<td><p><a class="reference internal" href="../../_images/vs_code_r_lsp.png"><img alt="vsc_rlsp" src="../../_images/vs_code_r_lsp.png" style="width: 200px;" /></a></p></td>
<td><p>The VS Code side of the R language server. Before installing this, run <code class="docutils literal notranslate"><span class="pre">install.packages(&quot;languageserver&quot;)</span></code>
inside an R prompt.</p></td>
</tr>
<tr class="row-even"><td><p>reStructuredText</p></td>
<td><p><a class="reference internal" href="../../_images/vs_code_rst.png"><img alt="vsc_rst" src="../../_images/vs_code_rst.png" style="width: 200px;" /></a></p></td>
<td><p>Enables reStructuredText support, the language used to write this documentation, among others.</p></td>
</tr>
<tr class="row-odd"><td><p>Jupyter</p></td>
<td><p><a class="reference internal" href="../../_images/vs_code_jupyter.png"><img alt="vsc_jupyter" src="../../_images/vs_code_jupyter.png" style="width: 200px;" /></a></p></td>
<td><p>Inline Jupyter notebook support. No more need to launch Jupyter in a web browser, just do it inside VS Code!</p></td>
</tr>
<tr class="row-even"><td><p>Code Spell Checker</p></td>
<td><p><a class="reference internal" href="../../_images/vs_code_spellcheck.png"><img alt="vsc_spellcheck" src="../../_images/vs_code_spellcheck.png" style="width: 200px;" /></a></p></td>
<td><p>Inline spell checker that is intelligent enough to not flag specific language-specific words, but still can
spell check comments and variable names.</p></td>
</tr>
</tbody>
</table>
</li>
<li><p><strong>RStudio:</strong> <em>(Optional)</em> If you don’t feel like using VS Code for your R work, the excellent, well-polished
standard IDE is RStudio Desktop, downloadable <a class="reference external" href="https://rstudio.com/products/rstudio/download/#download">here</a>.</p></li>
</ul>
</div>
</div>
</div>


           </div>
           
          </div>
          <footer>

  <hr/>

  <div role="contentinfo">
    <p>
        &#169; Copyright 2021, Galloway Lab. Shared under the Creative Commons 4.0 Attribution International license.
      <span class="lastupdated">
        Last updated on Jan 10, 2021.
      </span>

    </p>
  </div>
    
    
    
    Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    
    provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>
        </div>
      </div>

    </section>

  </div>
  

  <div class="rst-versions" data-toggle="rst-versions" role="note" aria-label="versions">
    <span class="rst-current-version" data-toggle="rst-current-version">
      <span class="fa fa-book"> Downloads and versions</span>
      v: latest
      <span class="fa fa-caret-down"></span>
    </span>
    <div class="rst-other-versions">
      
      
      <dl>
        <dt>Versions</dt>
        
           <strong> 
          <dd><a href="/protocols/en/latest/">latest</a></dd>
           </strong> 
        
          
          <dd><a href="/protocols/en/tech_wip/">tech_wip</a></dd>
          
        
          
          <dd><a href="/protocols/en//"></a></dd>
          
        
      </dl>
      
      
      <dl>
        <dt>Downloads</dt>
        
          <dd><a href="/protocols/en/latest/galloway_lab_protocols.pdf">pdf</a></dd>
        
      </dl>
      
      
      <hr/>
      Hosted with <span class="fa fa-heart"></span> on GitHub Pages.

    </div>
  </div>


  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script>

  
  
    
   

</body>
</html>